js/menu.js

```diff
@@ -848,3 +848,33 @@ document.addEventListener('DOMContentLoaded', function () {
   Toolbox.init();
   referencePane.init();
 })
+
+//add toggle icon to the PIN pane and add the click event handler to it
```

Comment:
Is there any way the PIN toggle could be added just like the current toggles are, so we don't have to duplicate the functionality?

Reply:
That's good thinking. I'll open a new PR after figuring it out.

Reply:
Please reuse existing PRs instead of creating clutter by opening duplicate ones :-/